verilog/tools/kythe/verilog_kythe_extractor.cc

```diff
@@ -154,6 +154,7 @@ static std::vector<absl::Status> ExtractFiles(
   if (absl::GetFlag(FLAGS_printextraction)) {
     std::cout << file_list_facts_tree << std::endl;
   }
+  LOG(INFO) << file_list_facts_tree << std::endl;
```

Comment:
remove (btw.  you can set --printextraction flag instead of always adding this line)